.github/ISSUE_TEMPLATE/BUG-REPORT.yml

```diff
@@ -0,0 +1,109 @@
+name: 🐞 Bug Report
+description: Report a bug
+labels: ['bug']
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Hi there! :wave: :smiley:
+        Thanks for taking the time to fill out this bug report!
+
+        > To expedite issue processing please search open and closed issues before submitting a new one.
+  - type: dropdown
+    id: package-concerned
+    attributes:
+      label: I am opening an issue for
+      options:
+        - Styleguide documentation
+        - Figma
+        - Icons
+        - Styles (CSS, SCSS, ...)
+        - Tokens
+        - Tools (PostCSS, Stylelint, ...)
```

Comment:
Change order:
- Documentation
- Design Tokens
- Tools (PostCSS, Stylelint, ...)
- Styles (CSS, SCSS, ...)
- Icon set
- Figma

Reply:
@tiloyi Should we add another option like "Component"?

Reply:
Fixed